coffeelint.json

```diff
@@ -27,5 +27,14 @@
   "braces_spacing": {
     "spaces": 0,
     "level": "error"
+  },
+  "spacing_after_comma": {
+    "level": "error"
+  },
+  "no_stand_alone_at": {
+    "level": "error"
+  },
+  "newlines_after_classes": {
+    "level": "error"
```

Comment:
Apparently it's a known issue: https://github.com/clutchski/coffeelint/issues/375